vignettes/epiprocess.Rmd

```diff
@@ -9,14 +9,56 @@ editor_options:
   chunk_output_type: console
 ---
 
-This package introduces a common data structure for epidemiological data sets
-measured over space and time, and offers associated utilities to perform basic
-signal processing tasks.
+The [`{epiprocess}`](https://cmu-delphi.github.io/epiprocess/) package provides
+
+- `epi_df`, a class for working with epidemiological time series data;
+- `epi_archive`, a class for working with the version history of such time series data;
+- sample data in these formats;
+- functions for common data transformations (e.g., 7-day averages);
+- functions for exploratory data analysis and situational awareness (e.g.,
+  outlier detection and growth rate estimation); and
+- functions for version-faithful "pseudoprospective" backtesting of models, and
+  other version history analysis.
+
+It is part of a broader suite of packages including
+[`{epidatr}`](https://cmu-delphi.github.io/epidatr/),
+[`{epidatasets}`](https://cmu-delphi.github.io/epidatasets/),
+[`{rtestim}`](https://dajmcdon.github.io/rtestim/), and
+[`{epipredict}`](https://cmu-delphi.github.io/epipredict/), for accessing,
```

Comment:
suggestion: reorder by importance/relevance to users. I might do:

```suggestion
It is part of a broader suite of packages that includes
[`{epipredict}`](https://cmu-delphi.github.io/epipredict/),
[`{epidatr}`](https://cmu-delphi.github.io/epidatr/),
[`{rtestim}`](https://dajmcdon.github.io/rtestim/), and
[`{epidatasets}`](https://cmu-delphi.github.io/epidatasets/), for accessing,
```

but then the following wording ("accessing, analyzing, and forecasting") isn't parallel anymore or if reordered isn't a logical flow. Not sure.

Reply:
Yeah, I struggled with the ordering here.  Think your suggestion is better despite the lack of parallelism (and difficulty restoring parallelism --- I gave up for now on trying).